modules/ROOT/pages/clauses/listing-functions.adoc

```diff
@@ -76,7 +76,7 @@ m| BOOLEAN
 
 [NOTE]
 ====
-More details about the syntax descriptions can be found xref:administration/index.adoc#administration-syntax[here].
+More details about the syntax descriptions can be found link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/syntax.adoc#administration-syntax-reading[here].
```

Comment:
and apparently that isn't the correct link either (gives page not found in preview)

Reply:
And fixed